tools/Utilities/StatusCheckLogger.cs

```diff
@@ -182,5 +200,43 @@ public async Task BuildCheckRunResult(string token, string owner, string repo, s
             Console.WriteLine("Exception details:");
             Console.WriteLine(e);
         }
+        try
+        { 
+            var core = _gitHubCoreService.Value;
+            
+            await core.GroupAsync("Writing run outputs", async () => 
+            {
+                await core.SetOutputAsync("check_name", title, JsonCheckRunAnnotationSerializerContext.Default.String);
+                
```

Comment:
Maybe group these into pairs of lines (so remove the blank line between the two check_name-related lines, and then between the two conclusion-related lines, etc.)